meerk40t/core/elements.py

```diff
@@ -4102,14 +4102,15 @@ def element_matrix(
             if len(data) == 0:
                 channel(_("No selected elements."))
                 return
-            if ty:
-                raise SyntaxError
```

Comment:
This syntax error might have wanted `if not ty` but it's better to just list the values when you type `matrix` without operands.